gdcdictionary/schemas/aggregated_somatic_mutation.yaml

```diff
@@ -44,39 +45,42 @@ required:
   - file_size
   - md5sum
   - data_category
-  - data_type
   - data_format
+  - data_type
   - experimental_strategy
   - somatic_aggregation_workflows
 
 uniqueKeys:
-  - [ id ]
-  - [ project_id, submitter_id ]
+  - [id]
+  - [project_id, submitter_id]
 
 properties:
-  $ref: "_definitions.yaml#/data_file_properties"
+
+  $ref:
+    - "_definitions.yaml#/data_file_properties"
 
   data_category:
-    term:
-      $ref: "_terms.yaml#/data_category"
-    enum: [ "Simple Nucleotide Variation" ]
+    $ref:
+      - "_terms.yaml#/data_category/common"
+    enum:
+      - Simple Nucleotide Variation
 
   data_type:
-    term:
-      $ref: "_terms.yaml#/data_type"
+    $ref:
+      - "_terms.yaml#/data_type/common"
```

Comment:
Took a closer look and realized that this is going to break the internal resolvers, will need a simple fix to support returning a list

Reply:
Is the cause of failed travis builds?  It's trying to call `split` on a `list`

Reply:
Yea, it was. Looks good now